templates/scripts/import-cf-keypair.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Imports the specified CF Keypair data into AWS SecretsManager.
+
+PROGNAME=$(basename "$0")
+
+function usage() {
+    echo "Usage: ${PROGNAME} [ -h | --help ] ID FILE"
+    echo
+    echo "ID: CloudFront Keypair ID (Access Key ID)"
+    echo "FILE: CloudFront Keypair private key file"
+    echo
+    echo "See: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs"
+}
+
+PARSED_ARGUMENTS=$(getopt -n ${PROGNAME} -o h --long help -- "$@")
+
+while : ; do
+    case "$1" in
+        -h | --help)
+            shift
+            usage
+            exit 0
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            break
+            ;;
+    esac
+done
+
+if [ "$#" -ne 2 ]; then
+    echo "Incorrect number of arguments"
+    exit 1
+fi
+
+ID=$1
+SECRET=$(tr -d '\r' < $2 | awk '{printf "%s\\n", $0}')
+
+aws secretsmanager \
+    create-secret \
+    --name <% .Name %>_cf_keypair \
+    --region <% index .Params `region` %> \
+    --secret-string "{\"keypair_id\":\"${ID}\",\"keypair_secret\":\"${SECRET}\"}"
```

Comment:
Maybe we can call this `private_key` instead of `keypair_secret`?

Reply:
Yeah, no prob. I'll update the node be repo to reflect that.